src/com/epam/javatrainings/classdecomposition/PalmettoPizzeriaCashDesk.java

```diff
@@ -0,0 +1,225 @@
+package com.epam.javatrainings.classdecomposition;
+
+import java.time.LocalTime;
+import java.util.*;
+
+public class PalmettoPizzeriaCashDesk {
```

Comment:
Arthur jan, do you feel comfortable while reading the code? I suggest you write each class in a separate source file, then submit for review. Thank you.